### PR TITLE
Update contributing and joining info

### PR DESCRIPTION
**Pull request by aphillips**
This PR is intended to work in tandem with @srl295's PR #875 

- change the contributing page to point to Unicode's contributing page
- change the contributing page to point to the README.md file for joining instructions
- change the README.md to include joining instructions consistent with Unicode's actual policy. Previously we just asked people to join our mailing list and watch the repo. That's good for _following_ the work.

### CODE REVIEW

**Comment by aphillips** [comment]
I definitely need to do something about delegates.md too, but am saving that for when I have time to do the work.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,32 +1,9 @@
...
-## Joining the Working Group
-
-We are looking for participation from software developers, localization engineers and others with experience
-in Internationalization (I18N) and Localization (L10N). If you wish to contribute to this work, please review
-the information on the Contributor License Agreement below. In addition, you should:
-
-1. Apply to join our [mailing list](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-wg)
-2. Watch this repository (use the "Watch" button in the upper right corner)
-
-<!-- boilerplate follows - do not edit -->
```
Comment (srl295) [inline CONTRIBUTING.md:12]: @aphillips note the 'do not edit' ^

**Review by srl295 — CHANGES_REQUESTED** [review]
Actually, you shouldn't point to the other CONTRIBUTING.md - copy and paste is the right thing here, because we need this text to be in front of users right away.  This file is special and GitHub presents it to the users as part of the PR process.

I guess alternately you could *delete* the CONTRIBUTING.md file and have the readme point to the .github one <https://github.com/unicode-org/.github/blob/main/.github/CONTRIBUTING.md> - that works also.  Then GitHub presents the 'community health file' from the special .github repo in place of the local CONTRIBUTING.md file.

File: CONTRIBUTING.md (hunk 1 above)
Comment (aphillips) [reply CONTRIBUTING.md:12]: I did. I removed it because I removed the boilerplate there.

**Review by annebright — APPROVED** [review]

**Comment by aphillips** [comment]
@srl295 I went with a version of (A), tweaking the joining info in CONTRIBUTING.md to point to the revised text in the readme.

**Review by srl295 — APPROVED** [review]

**Pull request merged**